packages/logger/src/logger.ts

```diff
@@ -15,6 +15,8 @@
  * limitations under the License.
  */
 
+import { LogCallback, LogLevelString, LogOptions } from '@firebase/app-types';
```

Comment:
It feels like these 3 types belong in the logger package and should be imported to `@firebase/app`

Reply:
Done.